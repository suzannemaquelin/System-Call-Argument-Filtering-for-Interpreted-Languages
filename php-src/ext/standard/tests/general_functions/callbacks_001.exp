parent|who
B
C|parent::who
B
B|parent::who
A
E|parent::who
D
A|who
A
C|who
B
B|who2
A
===FOREIGN===
parent|who
O
P|parent::who
O
$this|O::who
O
$this|B::who

Warning: call_user_func() expects parameter 1 to be a valid callback, class 'P' is not a subclass of 'B' in %s on line %d
===DONE===