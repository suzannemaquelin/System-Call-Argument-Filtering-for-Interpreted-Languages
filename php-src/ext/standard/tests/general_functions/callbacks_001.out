Failed loading /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so:  /opt/php-7.1/lib/php/extensions/debug-non-zts-20160303/xdebug.so: cannot open shared object file: No such file or directory
parent|who
B
C|parent::who
B
B|parent::who
A
E|parent::who
D
A|who
A
C|who
B
B|who2
A
===FOREIGN===
parent|who
O
P|parent::who
O
$this|O::who
O
$this|B::who

Warning: call_user_func() expects parameter 1 to be a valid callback, class 'P' is not a subclass of 'B' in /root/php-src/ext/standard/tests/general_functions/callbacks_001.php on line 63
===DONE===